src/guide/component-custom-events.md

```diff
@@ -141,7 +141,13 @@ app.component('user-name', {
 
 ## Handling `v-model` modifiers
 
-In 2.x, we have hard-coded support for modifiers like `.trim` on component `v-model`. However, it would be more useful if the component can support custom modifiers. In 3.x, modifiers added to a component `v-model` will be provided to the component via the modelModifiers prop:
+When we were learning about form input bindings, we saw that `v-model` has [built-in modifiers](/guide/forms.html#modifiers) like `.trim` or `.native`. In some cases, however, you might also want to add your own custom modifiers.
```

Comment:
My bad, I haven't caught it on the first round of review: `.native` does not exist anymore, so we should remove it

Reply:
Oops, yeah. Removed.